src/libraries/System.Speech/src/Internal/ObjectToken/RegistryDataKey.cs

```diff
@@ -484,12 +484,13 @@ private static string GetFirstKeyAndParseRemainder(ref string registryPath)
 
         private static RegistryKey RegKeyFromRootPath(string rootPath)
         {
-            RegistryKey[] roots = new RegistryKey[] {
+            ReadOnlySpan<RegistryKey> roots =
+            [
                 Registry.ClassesRoot,
                 Registry.LocalMachine,
                 Registry.CurrentUser,
                 Registry.CurrentConfig
-            };
+            ];
```

Comment:
Presumably this one doesn't allocate even before this change - https://godbolt.org/z/jq1fYPfxq

Reply:
It does.

Reply:
Actually, nvm, I see it now, I thought, it's Enum